backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -266,7 +267,20 @@ class LoggedInUserPasswordSchema(marshmallow.Schema):
 
 
 class SetConfigSchema(marshmallow.Schema):
-    parameters = marshmallow.fields.Dict(required=True, example='{"param1":"value1"}')
+    """
+    Change the user config
+    """
+
+    parameters = marshmallow.fields.Dict(
+        required=True,
+        example={"param1": "value1"},
+        validate=user_config_validator,
+        description="A simple json dictionary. "
+        'Valid key are limited to not empty string with "0-9a-zA-Z-_." characters. '
+        'You can use "." to create hierarchy in the configuration parameters. '
+        "Valid value allow only standard type: int, bool, null, float and do not accept "
+        "complex type such dict or list.",
```

Comment:
```suggestion
```suggestion
        'Valid keys only contain characters in "0-9a-zA-Z-_." and are not empty. '
        'You can use "." to create a hierarchy in the configuration parameters. '
        "Valid values only allow primitive types: numbers, bool, null, and do not accept "
        "complex types such dictionaries or lists.",
```